src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlTree.java

```diff
@@ -520,20 +520,18 @@ public static HtmlTree HTML(String lang, Content head, Content body) {
     }
 
     /**
-     * Creates an HTML {@code INPUT} element with the given id and initial value.
+     * Creates an HTML {@code INPUT} element with the given id.
      * The element as marked as initially disabled.
      *
      * @param type  the type of input
      * @param id    the id
-     * @param value the initial value
      * @return the element
      */
-    public static HtmlTree INPUT(String type, HtmlId id, String value) {
+    public static HtmlTree INPUT(String type, HtmlId id) {
         return new HtmlTree(TagName.INPUT)
                 .put(HtmlAttr.TYPE, type)
                 .setId(id)
-                .put(HtmlAttr.VALUE, value)
-                .put(HtmlAttr.DISABLED, "disabled");
+                .put(HtmlAttr.DISABLED, "");
```

Comment:
I'm curious: if such a call to `put` translates to a boolean HTML attribute, what call to `put` translates to an HTML attribute with an empty value?